st2common/st2common/runners/base.py

```diff
@@ -102,6 +102,12 @@ def pre_run(self):
     def run(self, action_parameters):
         raise NotImplementedError()
 
+    def pause(self):
+        raise NotImplementedError()
```

Comment:
Maybe we could update all those places where we throw `NotImplementedError` to include a message which is more user-friendly.

E.g.

```python
runner_name = getattr(self.runner_type_db, 'name', 'unknown')
raise NotImplementedError('"pause" operation is not supported for runner %s' % (runner_name)
```

Then we could also leverage this exception message higher up in the stack.